pkg/sensors/tracing/generickprobe_test.go

```diff
@@ -66,6 +66,17 @@ func Test_parseString(t *testing.T) {
 	})
 }
 
+func dummyValidateInfo(kprobes []v1alpha1.KProbeSpec) []*kpValidateInfo {
```

Comment:
nit, is it dummy? ;-) bit misleading to me because it actualy takes spec and fills in validate info, right?